docs/uma-tokenholders/uma-holders.md

```diff
@@ -0,0 +1,36 @@
+---
+title: UMA Tokenholder Responsibilities
+sidebar_label: Responsibilities
+---
+
+Owners of UMA tokens have 2 categories of responsibilities:
+
+1. Voting on price requests from financial contracts using the DVM
+2. Governing the UMA ecosystem by voting on parameter changes and approving system upgrades
+
+UMA tokenholders can use the [Voter dApp](https://vote.umaproject.org/) to vote on price requests. A tutorial on how to vote on price requests from the Voting dApp is [here](uma-tokenholders/voter-dApp.md). 
+
+Voters who participate and vote correctly earn an inflationary reward (currently 0.05% of total network token supply), distributed pro-rata by stake.
```

Comment:
Feel like we get a lot of confusion on this. Maybe we should give an example?

Reply:
Yep I like this change. Lets merge and do another PR next week on this.